src/Command/SymlinksCommand.php

```diff
@@ -135,6 +151,18 @@ private function symlinkThemes($rootDir, $output)
     }
 
     /**
+     * Generates a symlink relative to the given path
+     *
+     * @param string $path    The path to create symlink of
+     * @param string $rootDir The root directory
+     * @param string $output  The output object
+     */
+    private function relativeSymlink($path, $rootDir, OutputInterface $output)
```

Comment:
I don't think that would be clearer? There's already a method called `symlink`, and now one `createSymlink` :D